model/map/src/main/java/org/keycloak/models/map/realm/MapRealmAdapter.java

```diff
@@ -169,6 +169,17 @@ public void setRegistrationEmailAsUsername(boolean registrationEmailAsUsername)
         entity.setRegistrationEmailAsUsername(registrationEmailAsUsername);
     }
 
+    @Override
+    public boolean isEmailFormatAsUsernameAllowed() {
+        Boolean is = entity.isEmailFormatAsUsernameAllowed();
+        return Optional.ofNullable(is).orElse(false);
```

Comment:
Default setting seems to be `false`. Isn't that behavioral change from previous version? 

Let's imagine a situation where there are two different versions of keycloak in the same cluster, one without this change and one with it. Then it seems the resulting outcome will depend on the final location of the request, correct?

I believe the default behavior should match the one from previous version.